worker/provisioner/container_initialisation_test.go

```diff
@@ -368,13 +368,17 @@ func (t toolsFinderFunc) FindTools(v version.Number, series string, arch string)
 }
 
 func getContainerInstance() (cont []ContainerInstance, err error) {
+	pkgs := [][]string{
+		{"qemu-kvm"},
+		{"qemu-utils"},
+		{"genisoimage"},
+		{"libvirt-bin"},
+	}
+	if runtime.GOARCH == arch.ARM64 {
```

Comment:
This is actually here only for faking the container.Initialiser in this test suite.  We added more functionality to the KVM initialiser -- setup the libvirt disk pool -- which shouldn't get run. Adding that broke the container initialization So What this returns doesn't really matter, I'll make that more obvious.